client/state/jetpack-connect/actions.js

```diff
@@ -76,17 +76,19 @@ export default {
 		};
 	},
 
-	checkUrl( url, isUrlOnSites, flowType ) {
+	checkUrl( url, isUrlOnSites, selectedPlan ) {
 		return ( dispatch ) => {
 			if ( _fetching[ url ] ) {
 				return;
 			}
+			const action = {
```

Comment:
Just to be clear, you're talking about creating a new function (`createCheckUrlAction`) in the same module that is this:

```js
const createCheckUrlAction = ( url, selectedPlan ) => ( {
	type: JETPACK_CONNECT_CHECK_URL,
	selectedPlan,
	url,
} );
```

Is that correct?